packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -395,29 +419,33 @@ export const Account = ({
           </li>
         )}
 
-        <li className={styleClasses.li}>
-          <h3 className="mt-0 my-0 mb-1 text-lg">Connected Services</h3>
-        </li>
-        {attachedClients && attachedClients.length > 0 ? (
+        {guard.allow(AdminPanelFeature.ConnectedServices, user.group) && (
```

Comment:
Should we pull this logic into a guard component?

Reply:
Are you thinking just have a little wrapper component? If so I like that idea. The pattern in place is what Jody recommended, and I've seen similar things used for feature flags, but I can take a shot at the component approach instead. I must admit, sometimes all the curly braces and parens can be a bit much .

Reply:
Alright, I've extracted this into a Wrapper component. It looks much cleaner this way! Thanks for the advice.